api-server/src/events/events.controller.ts

```diff
@@ -37,6 +37,12 @@ import { WINSTON_MODULE_NEST_PROVIDER } from 'nest-winston';
 import { eventModelToEventDTO } from './dto/eventModelToEventDTO';
 import EventDTO from './dto/eventDTO';
 import { ENV } from '../constants';
+import { ApiImplicitQuery } from '@nestjs/swagger/dist/decorators/api-implicit-query.decorator';
```

Comment:
Is this how this is intended to be imported? Seems pretty messy by Nest's normal standards.

Reply:
this seems to be the only place to import it. I agree it's a bit of an odd path